terraform/aws/implementation/modules/rds/main.tf

```diff
@@ -0,0 +1,74 @@
+# Define the RDS instance for Postgres
+resource "aws_db_instance" "query-templates" {
+  identifier                = var.db_identifier
+  instance_class            = "db.t3.micro"
+  allocated_storage         = 5
+  engine                    = var.engine_type
+  engine_version            = var.engine_version
+  username                  = var.db_username
+  password                  = random_string.setup_rds_password.result
+  db_subnet_group_name      = aws_db_subnet_group.this.name
+  vpc_security_group_ids    = var.private_subnet_ids
+  parameter_group_name      = aws_db_parameter_group.this.name
+  publicly_accessible       = false
+  skip_final_snapshot       = true
+  final_snapshot_identifier = true
+  # backup_window             = var.backup_window
+  # maintenance_window        = var.maintenance_window
```

Comment:
Do these lines need an update?

Reply:
Good call!  I wasn't 100% sure if the VIPER team wanted automated `backup_window` and `maintenance_window` functionality for the db.  I confirmed with Dan it is not needed so I can definitely remove these attributes (and the variables).